R/constants.R

```diff
@@ -22,6 +22,8 @@ FLAG_XML <- "FLAG_XML"
 HISTORICAL_YEARS <- 1971:2010
 IMF_GDP_YEARS <- 2010:2020
 FUTURE_YEARS <- seq(2015, 2100, 5)
+BASE_YEARS <- c(1975, 1990, 2005, 2010)
```

Comment:
@bpbond According to Page's earlier comments "These are the model base years--not specific to the water module, and called by every single level2 code chunk. It should be called either MODEL_BASE_YEARS or GCAM_BASE_YEARS to clarify that it isn't the same as the historical years for data processing.", it is not water specific.